src/platform/ps4/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include "player.h"
+#include "output.h"
+
+static void log(LogLevel lvl, std::string const& message, LogCallbackUserData userdata)
+{
+
+}
+
+int main(int argc, char* argv[])
+{
+	std::vector<std::string> args(argv, argv + argc);
+	
+	args.push_back("--project-path");
+	args.push_back("/data/easyrpg-player");
+
+	Output::SetLogCallback(log, nullptr);
+	
+	Player::Init(std::move(args));
+	
+	Player::Run();
+	
+	while(1)
```

Comment:
I am not sure why you want to avoid exiting the player. Does the system crash or is returning to the menu not possible?

This "hang" and the changes in the gamebrowser are a bit intrusive I think.

Maybe we could show a fullscreen message here on how to reset the system or other instructions instead.